Bottom: d064a01efcd5ab68de48feaff6725d4ff4bd0cc0
Top:    0d446e5390746d812ba186cb3233366b0be4c260
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-01 23:45:55 +0000

Refresh of hid-gcode-add-include-for-loca

---

diff --git a/src/find.c b/src/find.c
index 16015dc..4c897ee 100644
--- a/src/find.c
+++ b/src/find.c
@@ -159,7 +159,7 @@ static bool IsRatPointOnLineEnd (PointType *, LineType *);
 static bool ArcArcIntersect (ArcType *, ArcType *);
 static bool PrepareNextLoop (FILE *);
 static void DrawNewConnections (void);
-///* XXX */ static bool SetThing (int, void *, void *, void *);
+/* XXX */ static bool SetThing (int, void *, void *, void *);
 static bool IsArcInPolygon (ArcType *, PolygonType *);
 static bool IsLineInPolygon (LineType *, PolygonType *);
 static bool IsPolygonInPolygon (PolygonType *, PolygonType *);
